packages/react-scripts/modernizr-config.json

```diff
@@ -0,0 +1,10 @@
+{
+    "minify": true,
+    "feature-detects": [
+      "test/customevent",
+      "test/dom/intersection-observer",
+      "test/es6/promises",
+      "test/network/fetch",
+      "test/url/parser"
```

Comment:
Looked at packages/react-scripts/polyfills.js and usage of these across apps to gather this list.

Reply:
You should move this comment into the json file in a special underscore comment key, or put it in the README so we know this information later on when we want to update it or understand why it's here